src/stockwatch/app/runtime/scraping.py

```diff
@@ -0,0 +1,9 @@
+"""Runtime data used for scraping the DeGiro site."""
```

Comment:
Why create this folder and file? Are you expecting more files here?
We have a method run_blocking which is not in this folder; we have another folder "run". And you named the file scraping, which we already had, I suggest to prevent duplicate names.

Doesn't it make sense to move this to degiro.py?

Reply:
Hmm I'm not so sure if I find the DeGiro.py file the correct destination. It's not really a use_case I would say.

Maybe a run/data.py file? Or rename run -> runtime . I think the current files in the run folder could be removed, I don't think we really need to support the cmdline run anymore.